tools/state-viewer/src/commands.rs

```diff
@@ -108,6 +111,26 @@ pub(crate) fn dump_state_redis(
     assert_eq!(res, Ok(()));
 }
 
+pub(crate) fn dump_tx(
+    height: BlockHeight,
+    home_dir: &Path,
+    near_config: NearConfig,
+    store: Store,
+    select_account_ids: Option<&Vec<AccountId>>,
+) -> Result<(), Error> {
+    let mut chain_store = ChainStore::new(
+        store.clone(),
+        near_config.genesis.config.genesis_height,
+        !near_config.client_config.archive,
+    );
+    let hash = chain_store.get_block_hash_by_height(height)?;
+    let block = chain_store.get_block(&hash)?;
+    let txs = tx_dump(&mut chain_store, &block, select_account_ids);
+    fs::write(PathBuf::from(&home_dir).join("tx.json"), json!(txs).to_string())
```

Comment:
the destination file should be a command line parameter (it can have a default value of 'tx.json')